docs/src/.vuepress/config.js

```diff
@@ -89,61 +85,68 @@ module.exports = {
           },
           {
             text: 'Permaweb',
-            link: '/concepts/permaweb'
-          },
-          {
-            text: 'Permaweb Applications',
-            link: '/concepts/permawebApplications'
+            link: '/concepts/permaweb',
+            collapsible: true,
+            children: [
+              {
+                text: 'Permaweb Applications',
+                link: '/concepts/permawebApplications',
+              }
+            ]
           },
           {
-            text: `Posting Transactions`,
-            link: `/concepts/post-transactions`,
+            text: 'SmartWeave',
+            link: '/concepts/smartweave',
             collapsible: true,
             children: [
               {
-                text: 'arweave-js',
-                link: '/guides/posting-transactions/arweave-js'
+                text: 'Arweave Name System (ArNS)',
+                link: '/concepts/arns'
               },
               {
-                text: 'bundlr.network',
-                link: '/guides/posting-transactions/bundlr'
+                text: 'Atomic Assets',
```

Comment:
I think they want to call these Atomic Tokens now, not Assets, the Assets is for the more broader def which is basically a Smart Contract

Reply:
oh. yep, that was just a habitual typo 😅